roles/keycloak/templates/16.1.0/standalone-ha.xml.j2

```diff
@@ -87,6 +87,12 @@
             <logger category="sun.rmi">
                 <level name="WARN"/>
             </logger>
+            <logger category="org.apache.xml.security.encryption.XMLCipher">
+                <level name="{{keycloak_logs_level.saml}}"/>
+            </logger>
+            <logger category="org.keycloak.saml.SAMLRequestParser">
+                <level name="{{keycloak_logs_level.saml}}"/>
```

Comment:
`{{ keycloak_log_level.saml }}` (i.e. change from plural and add spaces before brackets for consistency)